_observability-plugin/trace/trace-analytics-jaeger.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+title: Analyze Jaeger trace data 
+parent: Trace analytics
+nav_order: 55
+---
+
+# Analyze Jaeger trace data
+
+Introduced 2.5
+{: .label .label-purple }
```

Comment:
This doesn't seem to render out in the rich diff?

Reply:
see PDF for how it will render on the site. the Liquid variables don't render in the GitHub file viewer.